### DISCUSSION TITLE
Searching for original project (no remixes)

### DISCUSSION THREAD

**Discussion by sfederici** [ScratchAddons/ScratchAddons#5820] [Ideas] [answered]
A good Scratch project has usually a ton of remixes. Searching for an original project is not easy if you don't remember the author. My proposal is adding an "Original Projects" tab to the search results, where only original projects are shown.

**Comment by ScratchCat10090** [comment]
> 

- [ ] @1. > [_Butn****###``_](url)

**Comment by WorldLanguages** [comment, marked as answer]
Moved to new addon issue: #5825
